### ISSUE TITLE
Problems in the Wii version

### ISSUE THREAD

**Issue by Edu789** [libretro/fbalpha2012_neogeo#53]
In the last nightly build, the Wii version doesn’t work, and when it was working, the classic control didn’t work, in version 1.8.4 it was working perfectly with the core, since then the core started having problems until today...

**Comment by inactive123** [comment]
Can you perhaps ask @Wiimpathy if we can look at this together again?

**Comment by Edu789** [comment]
This version of Retroarch 1.8.4 was the last version of the Neo Geo core that worked well ...
https://github.com/Edu789/Edu/blob/ee2f876433d73d0c642c336abb069dc52027927d/fbalpha2012_neogeo_libretro_wii.dol

**Comment by Cee123** [comment]
> 
> 
> This version of Retroarch 1.8.4 was the last version of the Neo Geo core that worked well ...
> https://github.com/Edu789/Edu/blob/ee2f876433d73d0c642c336abb069dc52027927d/fbalpha2012_neogeo_libretro_wii.dol

That was before the whole build bot got hacked and taken down. I noticed that after that happened, there was a tonne of different issues with different cores. Some have been fixed and others are still broken.

**Comment by Edu789** [comment]
> > This version of Retroarch 1.8.4 was the last version of the Neo Geo core that worked well ...
> > https://github.com/Edu789/Edu/blob/ee2f876433d73d0c642c336abb069dc52027927d/fbalpha2012_neogeo_libretro_wii.dol
> 
> That was before the whole build bot got hacked and taken down. I noticed that after that happened, there was a tonne of different issues with different cores. Some have been fixed and others are still broken.

Ok, the vice cores are still locking on the black screen?

**Comment by Cee123** [comment]
> 
> 
> > > This version of Retroarch 1.8.4 was the last version of the Neo Geo core that worked well ...
> > > https://github.com/Edu789/Edu/blob/ee2f876433d73d0c642c336abb069dc52027927d/fbalpha2012_neogeo_libretro_wii.dol
> > 
> > 
> > That was before the whole build bot got hacked and taken down. I noticed that after that happened, there was a tonne of different issues with different cores. Some have been fixed and others are still broken.
> 
> Ok, the vice cores are still locking on the black screen?

Yeah those don't work at all. They just freeze upon loading the cores.
